tasks/setup-mysql.yml

```diff
@@ -1,9 +1,14 @@
 ---
+- name: install MySQL-Client
```

Comment:
ci fails with this task, `mysql package not found` you probably intended to install `python-mysql`?

Reply:
New commits added to support both ubuntu/debian and centos/redhat.